core/core-workflow-steps/src/main/java/org/eclipse/sw360/antenna/workflow/generators/SourceZipWriterImpl.java

```diff
@@ -131,12 +135,6 @@ private void addContentToZip(Artifact artifact, ZipOutputStream zipOut) {
             try {
                 LOGGER.debug("Writing File: {}", entryName);
                 writeContentToZipEntry(zipOut, sourceFile.get().toFile(), entryName);
-            } catch (ZipException e) {
-                if (!e.getMessage().equals("zip file is empty")) {
-                    reporter.add(artifact, MessageType.PROCESSING_FAILURE,
-                            e.getMessage() + ": caused by File: " + sourceFile);
-                    LOGGER.warn("{}: caused by File: {}", e.getMessage(), sourceFile);
-                }
```

Comment:
There was some special behavior for empty Zip files. Has it been tested that Commons Compress behaves in the same way here? Just asking because I am a bit sceptic regarding the existing test coverage.

Reply:
I was referring to the code fragment in the catch clause that has been deleted. Here exceptions with the message "zip file is empty" are obviously ignored; so I would assume that such files have caused problems in the past. It is pretty sure that Commons Compress will use different exception messages.

Reply:
I see what you mean. I wrote a test with an empty file. Apache compress indeed throws a ZipException. However, the message is different. Instead of "zip file is empty" it says, "Archive is not a ZIP archive". Do you think we should just change this message? I am not quite sure, since the latter message could also mean other things than an empty file.

Reply:
Checking for an error message is a pretty fragile approach. I am also not sure about the background of this check. So, some content is added to an archive, and afterwards an exception is ignored that there is no content? Maybe it is possible to change the workflow: Check if there is content to add to the ZIP, log a message if not, otherwise write the archive and treat all errors as critical.

Reply:
I also would want to know, if during a run a zip file was empty. We could leave it as it is now, since every IOException (a ZipException is derive from it to) is caught, logged and added to the reporter.

Reply:
Ok, agreed.